frontend/packages/dev-console/integration-tests/support/pages/topology/topology-page.ts

```diff
@@ -126,41 +138,61 @@ export const topologyPage = {
       .should('be.visible')
       .click(),
   rightClickOnNode: (releaseName: string) => {
-    cy.get('g.odc-base-node__label')
+    cy.get(topologyPO.graph.node)
       .should('be.visible')
       .contains(releaseName)
       .trigger('contextmenu', { force: true });
   },
   clickOnNode: (releaseName: string) => {
-    cy.get('g.odc-base-node__label')
+    cy.get(topologyPO.graph.node)
       .should('be.visible')
       .contains(releaseName)
       .click({ force: true });
   },
   clickOnSinkBinding: () => {
-    cy.get('g.odc-base-node__label')
+    cy.get(topologyPO.graph.node)
       .should('be.visible')
       .contains('sink-binding')
       .click({ force: true });
   },
   rightClickOnKnativeRevision: () => {
-    cy.byLegacyTestID('base-node-handler')
+    cy.get(topologyPO.graph.revisionNode)
       .find('g.odc-resource-icon')
       .trigger('contextmenu', { force: true });
   },
   clickOnKnativeRevision: () => {
-    cy.byLegacyTestID('base-node-handler')
+    cy.get(topologyPO.graph.revisionNode)
       .find('g.odc-resource-icon')
       .click({ force: true });
   },
   waitForKnativeRevision: () => {
-    cy.get('[data-test-id="base-node-handler"]', { timeout: 300000 }).should('be.visible');
+    cy.get(topologyPO.graph.revisionNode, { timeout: 300000 }).should('be.visible');
   },
   rightClickOnHelmWorkload: () => {
-    cy.byLegacyTestID('base-node-handler')
+    cy.byLegacyTestID('base-node-handler');
+    cy.get(topologyPO.graph.revisionNode)
       .find('circle')
       .trigger('contextmenu', { force: true });
   },
+  clickOnHelmWorkload: () => {
+    cy.get(topologyPO.graph.revisionNode)
```

Comment:
for `clickOnHelmWorkload` why we looking for `topologyPO.graph.revisionNode` . Is this knative Revision?

Reply:
topologyPO.graph.revisionNode is same for all workloads

Reply:
isn't that confusing as name `revisionNode` implies serverless revision node?

Reply:
Yes, valid point, updated the name